raiden-dapp/src/locales/en.json

```diff
@@ -10,7 +10,7 @@
       "no-checksum": "Please enter an address in checksum format. You can convert your address on https://ethsum.netlify.com",
       "empty": "Please enter an address or ENS name",
       "ens-resolve-failed": "Please enter a resolvable ENS name",
-      "invalid-excluded-address": "Please enter a different address or ENS name. You can't use your own address or the address of the token network."
+      "invalid-excluded-address": "Please enter a different address or ENS name, because the address you have entered is not allowed. There can be multiple reasons: <br />1) You can't use your own address <br />2) You can't use the address of the token network <br />3) The channel to this hub address isn't open"
```

Comment:
Checking the screenshot and seeing this text gives me a feeling that we are heading towards the wrong direction with this. We should try to keep the error messages for the validation short and descriptive. Usually, the message should fit into a single like underneath the input element.

Reply:
Feel free to create a follow up ticket for this 👍

Reply:
I like the `blocked` approach better. We certainly don't have to aggregate everything. Can you add this as part of this PR?